compiler/src/main/kotlin/io/spine/protodata/plugin/Plugin.kt

```diff
@@ -61,6 +69,9 @@ public interface Plugin {
  */
 internal fun BoundedContextBuilder.apply(plugin: Plugin) {
     plugin.views.forEach {
+        add(ViewRepository.default(it))
+    }
+    plugin.viewRepositories.forEach {
```

Comment:
What happens if we add a `View` and then a custom `Repository` which manages the same kind of view?